.github/workflows/ci.yml

```diff
@@ -32,22 +36,14 @@ jobs:
             ${{ runner.os }}-pip-
             ${{ runner.os }}-
 
-      - name: Add libtorrent deb repository
-        uses: myci-actions/add-deb-repo@8
-        with:
-          repo: deb http://ppa.launchpad.net/libtorrent.org/1.2-daily/ubuntu focal main
-          repo-name: libtorrent
-          keys: 58E5430D9667FAEFFCA0B93F32309D6B9E009EDB
-          key-server: keyserver.ubuntu.com
-          install: python3-libtorrent-dbg
-
       - name: Sets env var for security
         if: (github.event_name == 'pull_request' && contains(github.event.pull_request.body, 'security_test')) || (github.event_name == 'push' && contains(github.event.head_commit.message, 'security_test'))
         run: echo "SECURITY_TESTS=True" >> $GITHUB_ENV
 
       - name: Install dependencies
         run: |
           pip install --upgrade pip wheel
+          pip install libtorrent==1.2.*
```

Comment:
let's move this to the requirements.txt and remove the version requirement

Reply:
Just a reminder, removing the version requirement will install libtorrent 2